control/control.xml

```diff
@@ -390,12 +390,26 @@ textdomain="control"
         <software>
           <default_patterns>kde x11 base enhanced_base x11_yast yast2_basis</default_patterns>
         </software>
+        <partitioning>
+          <proposal>
```

Comment:
Is it going to be used for all the roles? If so, why not changing it only once in the partitioning section?